_build/conf.py

```diff
@@ -108,6 +109,7 @@
 lexers['rst'] = RstLexer()
 lexers['varnish3'] = CLexer()
 lexers['varnish4'] = CLexer()
+lexers['terminal'] = TerminalLexer()
```

Comment:
@javiereguiluz you need to add this to the `conf.py` used by the doc builder. You probably also have to add `sys.path.append(os.path.abspath('_theme/_exts'))` (where `os.path.abspath('_theme/_exts')` should be replaced by the path to the `_theme/_exts` directory)